src/sentry/integrations/repository/issue_alert.py

```diff
@@ -1,12 +1,39 @@
 from __future__ import annotations
 
+from dataclasses import dataclass
 from logging import Logger, getLogger
 
+from sentry.integrations.repository.base import BaseNotificationMessage
 from sentry.models.notificationmessage import NotificationMessage
+from sentry.models.rulefirehistory import RuleFireHistory
 
 _default_logger: Logger = getLogger(__name__)
 
 
+@dataclass(frozen=True)
+class IssueAlertNotificationMessage(BaseNotificationMessage):
+    # TODO(Yash): do we really need this entire model, or can we whittle it down to what we need?
```

Comment:
nit: I saw a bunch of these so far. Can you create a ticket so if in future someone else owns this know what the plan is to figure these out and refactor? Bonus point for adding ticket number to comment.

Reply:
Yeah let me add to the backlog!

Reply:
Done! https://github.com/getsentry/sentry/issues/66751

Reply:
updated the comments too :D